### PR TITLE
doc: Create self-assessment first version #220

### PR DESCRIPTION
**Pull request by alvarobernal2412**
# Description
A self-assessment of sprints 1 and 2 has been carried out on the basis of the failure conditions of the subject, justifying each score.

# Checklist:
- [x] I have performed a self-review of my code running `npm run build` and `npm run serve` after pushing the code.
- [x] I have checked that there is no fragile information published unintentionally.

# Screenshot:
Captura de pantalla 2024-04-01 a las 10 46 44
Captura de pantalla 2024-04-01 a las 10 46 52

### CODE REVIEW

**Review by alejandromd — APPROVED** [review]
This is really perfect. You take into account everything we've been talking about as a team so far in the project.

File: docs/Autoevaluación/Sprint 1.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+| Requisitos Potenciales           | Indicar requisitos potenciales para utilizar el sistema, como activación de ubicación u otros.               | [x] Sí [ ] No |  10 | Además, en el docusaurus tenemos un apartado de casos de uso en los que se mencionan estos casos de uso y la ONG a la que aplica.                                                                                                                                                                                                                                                              |
+
+#### Condiciones Suficientes para el Fracaso del Software
+En este apartado, en caso de que NO se cumpla la condición de fallo se marcará la columna de Sí.
+
```
Comment (AntonioRodriguezRuiz) [inline docs/Autoevaluación/Sprint 1.md:37]: This can be confusing to understand

**Review by AntonioRodriguezRuiz — CHANGES_REQUESTED** [review]

File: docs/Autoevaluación/Sprint 1.md (hunk 1 above)
Comment (alvarobernal2412) [reply docs/Autoevaluación/Sprint 1.md]: Done! Thank you for your review!

**Review by AntonioRodriguezRuiz — APPROVED** [review]
LGTM

**Pull request merged**